phantomgames/GameDetails/GameDetailRepository.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  GameDetailRepository.swift
```

Comment:
can delete this empty file if not in use?

Reply:
Please note that these files are here so that I could create the features segues but the feature development is in the next pull request

Reply:
okay. But please put warnings then. This was all communicated to do so in the PR guidelines and standards